spec/controllers/account_activations_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe AccountActivationsController do
+  describe '#edit' do
+    context 'with inactive user' do
+      let(:artist) { FactoryGirl.create(:artist) }
```

Comment:
Oh! I forgot to mention, using rspec and having a clearer connection between code and tests also makes it easier for tools like [guard](https://github.com/guard/guard-rspec) to run tests only related to the specific code you just changed and give you quicker feedback on if your code matches specs.